editor/scene_tree_dock.cpp

```diff
@@ -942,6 +942,18 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				break;
 			}
 
+			if (tocopy->get_owner() != scene) {
+				accept->set_text(TTR("Can't save a branch which is the child of an already instantiated scene. \nTo save this branch into its own scene, open the original scene, right click on this branch, and select 'Save Branch as Scene'."));
```

Comment:
```suggestion
				accept->set_text(TTR("Can't save a branch which is a child of an already instantiated scene. \nTo save this branch into its own scene, open the original scene, right click on this branch, and select 'Save Branch as Scene'."));
```
Also probably
```suggestion
				accept->set_text(TTR("Can't save a branch which is the child of an already instantiated scene. \nTo save this branch into its own scene, open the original scene, right click on this branch, and select Save Branch as Scene."));
```
(seeing as above there are no quotes)

Reply:
I'd keep the quotes, though maybe making them double (`\"Save Branch as Scene\"`).